quiz.css

```diff
@@ -0,0 +1,69 @@
+/*Block Styling and Positioning*/
+.medium,
+.dark,
+.tshirt,
+.green {
+  border:1px solid lightblue;
+  margin: 20px 10px;
+  max-width: 30%;
+  display: inline-block;
+}
+
+/*Inside the box styling*/
+
+h1 {
+  border:1px solid lightgray;
+  text-align: center;
+  line-height: normal;
+}
+
+h4 {
+  line-height: 4px;
+  font-weight: normal;
+}
+
+img {
+  height:130px;
+  width:100px;
+  margin: 0 auto;
+  display: block;
+}
+
+p {
+  margin: 20px 10px 20px 10px;
+}
+
+.green img {
+  min-height: 130px;
+  width: 150px;
+}
+
+.tshirt img {
+  height: 100px;
+  width: 150px;
+}
+
+.description {
+  margin-top: 10px;
+  text-align: center;
+}
```

Comment:
Under the Style Requirements you it specifies that the descriptions should be justified, not centered.
Also that the title and product image are centered. (Which implies that other parts wouldn't be centered, as shown in the example given.)